api/src/shared/application/usecases/checkUserOwnsCertificationCourse.js

```diff
@@ -2,7 +2,7 @@ import * as certificationCourseRepository from '../../../certification/shared/in
 
 const execute = async function ({ userId, certificationCourseId, dependencies = { certificationCourseRepository } }) {
   const certificationCourse = await dependencies.certificationCourseRepository.get({ id: certificationCourseId });
-  return certificationCourse.doesBelongTo(userId);
+  return certificationCourse._userId === userId;
```

Comment:
Pourquoi un underscore ? Je ne sais pas

Reply:
As-tu eu la reponse a cette question ?

En tout cas pour moi l'underscore c'est l'ancienne notation pour preciser un attribut prive (desormais ca devrait etre du `#userId`).

Par convention cela m'indique generalement de rechercher un getter associe, sans quoi je bypass les intentions des auteur du code originel, et du fait il existe pour le userId, je propose donc de l'utiliser ici

```suggestion
  return certificationCourse.getId()=== userId;
```